external-crates/move/crates/move-compiler/src/parser/token_set.rs

```diff
@@ -171,6 +171,12 @@ pub static VALUE_START_SET: Lazy<TokenSet> = Lazy::new(|| TokenSet::from(VALUE_S
 //**************************************************************************************************
 
 #[allow(dead_code)]
+impl Default for TokenSet {
```

Comment:
NOOOOO don't listen to clippy! This is a terrible lint rule, and IMO we should turn it off....